src/sass/list/stories/list.stories.js

```diff
@@ -0,0 +1,331 @@
+export default { title: "Skin/List" };
+
+export const base = () => `
```

Comment:
Can you format these? If you add `/* HTML */` (must be uppercase) before the template literal Prettier will do it for you